dataactcore/migrations/versions/25b8f47e075c_add_length.py

```diff
@@ -0,0 +1,76 @@
+"""add_length
+
+Revision ID: 25b8f47e075c
+Revises: 61dbcecb7ca1
+Create Date: 2016-07-08 11:05:22.608000
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '25b8f47e075c'
+down_revision = '1d0ef8e21ddd'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+
+
+def upgrade_error_data():
+    ### commands auto generated by Alembic - please adjust! ###
+    pass
+    ### end Alembic commands ###
+
+
+def downgrade_error_data():
+    ### commands auto generated by Alembic - please adjust! ###
+    pass
+    ### end Alembic commands ###
+
+
+def upgrade_job_tracker():
+    ### commands auto generated by Alembic - please adjust! ###
+    pass
+    ### end Alembic commands ###
+
+
+def downgrade_job_tracker():
+    ### commands auto generated by Alembic - please adjust! ###
+    pass
+    ### end Alembic commands ###
+
+
+def upgrade_user_manager():
+    ### commands auto generated by Alembic - please adjust! ###
+    pass
+    ### end Alembic commands ###
+
+
+def downgrade_user_manager():
+    ### commands auto generated by Alembic - please adjust! ###
+    pass
+    ### end Alembic commands ###
+
+
+def upgrade_validation():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('file_columns', sa.Column('length', sa.Integer(), nullable=True))
```

Comment:
Is the goal to handle `NULL` values from the database or do we want to default to `0` for the length column?